blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricDataQueryServer.java

```diff
@@ -79,6 +80,9 @@ public void startServer() {
                         Executors.newFixedThreadPool(workerThreads)));
         server.setPipelineFactory(new MetricsHttpServerPipelineFactory(router));
         serverChannel =  (ServerChannel) server.bind(new InetSocketAddress(httpQueryHost, httpQueryPort));
+
+        log.info("Starting tracker service");
+        Tracker.register();
```

Comment:
I did it this way to avoid those class having to declare and instantiate a Tracker object without having to use it for anything.  The code looks cleaner this way.